### ISSUE TITLE
Make it possible to delete users

### ISSUE THREAD

**Issue by hennevogel** [openSUSE/osem#383]
- [x] agree on method of deletion in #330 
- [x] change the destroy function in users controller
- [x] fix for authorization in the users view page
- [x] fix the migration in #333

**Comment by ChrisBr** [comment]
@differentreality 
Isn't that already implemented resp. what needs to be done?

**Comment by gopesht** [comment]
Yes,I think this needs to be closed, we have delete feature in the Users Page for admins.

**Comment by differentreality** [comment]
@ChrisBr check https://github.com/openSUSE/osem/blob/master/app/controllers/admin/users_controller.rb#L39

**Comment by gopesht** [comment]
Well I didn't see that, thanks @differentreality .I think some work was done, it got reverted back.This thing needs to be fixed ASAP otherwise there will be Db issues in production like we had earlier.
